### ISSUE TITLE
Cannot create a new dimension

### ISSUE THREAD

**Issue by TheVanoq** [McJty/RFToolsDimensions#108]
I cannot create a new dimension. The tab gets "realized", the games hangs for a second and I cannot dial it.
Right now, the mod is completely unplayble since this happens with every dimension I try to create.

Versions:
```
Minecraft: 1.10.2
Forge: 1.10.2-12.18.3.2297
rftools-1.1x-6.01.jar
rftoolsctrl-1.1x-1.6.8.jar
rftoolsdim-1.1x-5.02.jar
mcjtylib-1.1x-2.3.11.jar
```

Error: https://pastebin.com/mvUHtz9k
Another: https://pastebin.com/0YDXaiY5

**Comment by McJty** [comment]
I need your full log. This is a much bigger log with more info in

Also what version of compatlayer?

**Comment by TheVanoq** [comment]
`compatlayer-1.10-0.2.9.jar`

Check line 19855: https://www.thevanoq.com/download/fml-server-latest.log

**Comment by McJty** [comment]
Ok. I see what's wrong. Somehow you got a single controller dimension but no biome specified. So it cannot find a biome. How did you make this dimension? Was it random or did you use specific dimlets?

**Comment by TheVanoq** [comment]
Completely random. It seems to work fine in singleplayer but not in multiplayer.

**Comment by McJty** [comment]
You mean every dimension is like that? You use different dimlet sequences everytime? (otherwise you get the same dimension). What is your server exactly? What kind of mods?

**Comment by TheVanoq** [comment]
Yes every dimension is broken. I keep safedeling them and then create new, random ones. None work. I cannot post the mod list right now since I'm not at my desktop until tomorrow.

**Comment by McJty** [comment]
You mean create new random ones with DIFFERENT dimlets? That's important. Otherwise you will get exactly the same one

**Comment by TheVanoq** [comment]
Server is the usual Minecraft Forge Server running on Ubuntu 16.04.
The modlist: https://pastebin.com/sFWN9r26

I create new random ones without specifying ANY kind of attributes.
It should fill it with random ones then, right? I just put empty dimension tabs in and build it using the Dimension Builder.

**Comment by TheVanoq** [comment]
Creating an "empty" dimension without any attributes specified for it causes this issue.
Adding random "digit" attributes to it seems to fix it and I am able to enter the world.

Also, there is a bug with the safedel command. Sometimes it cannot delete the world because it "is still in use" even though there is no player inside, the dimension has no power and the dimension tab referencing it has been cleared and deleted. Do you want me to create a separate issue for that?
